examples/run_gesture_recognition.py

```diff
@@ -59,6 +60,7 @@
         sense.display.DisplayFPS(expected_camera_fps=net.fps,
                                  expected_inference_fps=net.fps / net.step_size),
         sense.display.DisplayTopKClassificationOutputs(top_k=1, threshold=0.5),
+        sense.display.DisplayClassnameOverlay(thresholds=LAB_THRESHOLDS),
```

Comment:
The `border_size` used in `DisplayResults` is different depending on if a `title` is given. Therefore, the text isn't currently always perfectly centered.

Reply:
Hmm, any suggestion here? Shall we add title on top first, let `display_op` inherit `DisplayResults`'s border size, and then display information?

Reply:
Maybe take a look at how I did it in `run_digit_recognition`: https://github.com/TwentyBN/sense/pull/64/files
I'm not perfectly happy with it, but it works for now.